### ISSUE TITLE
Longitude and latitude of stations

### ISSUE THREAD

**Issue by zaninip** [c9s/soramame#1]
Is there a way to get latitude and longitude of the stations? If not, it would be a very useful feature.
Thanks in advance for the answer.

**Comment by c9s** [comment]
@zaninip yes. however some stations in JP might not support the latitude and longitude.
